config/chassis_modules.py

```diff
@@ -110,8 +110,9 @@ def shutdown_chassis_module(db, chassis_module_name):
 
     if not chassis_module_name.startswith("SUPERVISOR") and \
        not chassis_module_name.startswith("LINE-CARD") and \
-       not chassis_module_name.startswith("FABRIC-CARD"):
-        ctx.fail("'module_name' has to begin with 'SUPERVISOR', 'LINE-CARD' or 'FABRIC-CARD'")
+       not chassis_module_name.startswith("FABRIC-CARD") and \
+       not chassis_module_name.startswith("DPU"):
```

Comment:
For shutdown method, do we have to follow same process as mentioned in https://github.com/sonic-net/SONiC/pull/1699?

Reply:
yes, the config change handler triggers the same code.